pkg/daemon/ceph/osd/encryption.go

```diff
@@ -160,6 +162,16 @@ func dumpLUKS(context *clusterd.Context, disk string) (string, error) {
 	return cryptsetupOut, nil
 }
 
+func openLUKS(context *clusterd.Context, disk, target, passphrase string) error {
+	args := []string{"luksOpen", "--verbose", "--allow-discards", disk, target}
+	err := context.Executor.ExecuteCommandWithStdin(luksOpenCmdTimeOut, cryptsetupBinary, &passphrase, args...)
```

Comment:
cryptsetup has a timeout option already:

```
--timeout, -t
The number of seconds to wait before timeout. This option is relevant every time a password is asked, like create, luksOpen, luksFormat or luksAddKey. It has no effect if used in conjunction with --key-file.
```

So let's use that instead?

Reply:
that is to wait for stdin only.

current timeout will kill the entire process, even if it gets stuck after stdin is taken.

Reply:
Makes sense to keep Rook's internal timeout then.